ttax/decompositions.py

```diff
@@ -0,0 +1,181 @@
+import numpy as np
+import jax
+import jax.numpy as jnp
+
+from ttax.base_class import TT
+from ttax.base_class import TTMatrix
+
+def tt_round(tt, max_tt_rank=None, epsilon=None, name='ttax_round'):
+  """TT-rounding procedure, returns a TT object with smaller TT-ranks.
+  Args:
+    tt: `TensorTrain` object, TT-tensor or TT-matrix
+    max_tt_rank: a number or a list of numbers
+      If a number, than defines the maximal TT-rank of the result.
+      If a list of numbers, than `max_tt_rank` length should be d+1
+      (where d is the rank of `tens`) and `max_tt_rank[i]` defines
```

Comment:
Better say not "rank" of the tens but maybe something like "number of dimensions" to avoid confusing it with TT-rank